packages/workflows-sdk/src/utils/composer/type.ts

```diff
@@ -10,6 +10,15 @@ import { Context, MedusaContainer } from "@medusajs/types"
 export type StepFunctionResult<TOutput extends unknown | unknown[] = unknown> =
```

Comment:
@adrien2p, it seems these changes are possibly causing some issues for the Redis workflow engine tests. See [here](https://github.com/medusajs/medusa/actions/runs/8021017524/job/21912240155?pr=6449). Does that sound plausible? Can't see what else in the PR should cause it.

Reply:
yes I ve sent you a message the otherday that some tests will fail and need fixing, mainly becuse of symbole vs string. Do you want me to jump on it after i finish my thing?

Reply:
I thought you referred to tests in the context of the cart module. I don't think I am in a position to solve the workflow-related issues, so that would be great - thanks 💪